### PR TITLE
project importing

### PR DESCRIPTION
**Pull request by giorgioprovenzale**

### CODE REVIEW

File: advanced/project-importing.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# Project Importing
+
+Mimic app can import projects from external file, this feature could be very useful and it is very easy to use.
```
Comment (MihalyBezzeg) [inline advanced/project-importing.md:3]: I'd add an example here to explain why this feature is useful

File: advanced/project-importing.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+Mimic doesn't import the file but only save a reference to it, so the path of the file used in this process must not change. For instance this feature if very useful is you save the JSON file with the project settings in the same folder of your project
```
Comment (MihalyBezzeg) [inline advanced/project-importing.md]: only save**s** a reference must not **be** change**d**

Comment (MihalyBezzeg) [inline advanced/project-importing.md]: The second part of the paragraph: _For instance this feature if very useful is you save the JSON file with the project settings in the same folder of your project_ is not clear to me. Could you please explain?

File: advanced/project-importing.md (hunk 1 above)
Comment (giorgioprovenzale) [reply advanced/project-importing.md:3]: I added an example on the next paragraph, maybe I can explain that with the file in project folder you can share it through GIT with other team members

File: advanced/project-importing.md (hunk 2 above)
Comment (giorgioprovenzale) [reply advanced/project-importing.md]: I will explain it better, the idea is that you can share this file with other members of the team with git. So everyone has the same config file for Mimic. I will fix it

File: advanced/project-importing.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+_Endpoint_
+
+Each endpoint has _uuid_, _path_, _method_, _response_, _request_, _statusCode_, _timeout_,
+
+- The _uuid_ must be unique, for instance, use the project name, the endpoint path and some number like in the example.
+- The _path_ is the path of the endpoint, like `/users`.
+- The _method_ is the method of the endpoint, like `GET`, `POST`, `PUT`, `DELETE`, `PATCH`.
+- The _response_ is the response of the endpoint, here you can set the JSON used as response.
+- The _request_ is the request of the endpoint, here you can set the _body_ (JSON) and the _parameters_ of the request. Like `sortby=date`.
```
Comment (MihalyBezzeg) [inline advanced/project-importing.md]: I think the ? mark is missing from params: ?sortby=date

File: advanced/project-importing.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+![Import Project Info](../Images/import_project_info.png 'Import Project Info')
+
+Your project is imported!
+
+You can find it in the projects list:
+
+![Imported Project Menu](../Images/imported_project_menu.png 'Imported Project Menu')
+
+settingsand clicking on it you will find all endpoints set on the JSON file:
```
Comment (MihalyBezzeg) [inline advanced/project-importing.md]: I think the start of the sentence is missing

**Review by MihalyBezzeg — APPROVED** [review]

**Pull request merged**